File: Python/dynamo.py
t = int(input())
for i in range(t):
    n = int(input())
    a = int(input())
    s = 10**n + 10**n + a
    print(s)
    b = int(input())
    c = 10**n - b
    print(c)
    d = int(input())
    e = 10**n - d
    print(e)
    grade = int(input())
    if grade == -1:
        break  

File: CPP/dynamo.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int t;
    cin >> t;
    while(t--)
    {
        int n, grade;
        unsigned long long a, s, b, c, d, e;
        cin >> n >> a;
        s = pow(10, n)*2 + a;
        cout << s << endl;
        cin >> b;
        c = pow(10, n) - b;
        cout << c << endl;
        cin >> d;
        e = pow(10, n) - d;
        cout << e << endl;
        cin >> grade;
        if(grade == -1)
        break;
    }
    return 0;
}